docs/BuildConfiguration.md

```diff
@@ -88,7 +90,7 @@ The netexDefaults section of `build-config.json` allows you to specify default p
 ## OSM 
 
 The osm section of `build-config.json` allows you to override the default behavior of scanning
-for OpenStreetMap files in the [base directory](Configuration.md#Base Directory). You can specify data
+for OpenStreetMap files in the [base directory](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/docs/Configuration.md#Base-Directory). You can specify data
```

Comment:
Do these types of links work nicely with read the docs on http://docs.opentripplanner.org/en/latest/ ?